WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -70,4 +72,23 @@ + (NSArray *)routes
   return FBResponseWithOK();
 }
 
++ (id<FBResponsePayload>)handleDismissKeyboardCommand:(FBRouteRequest *)request
+{
+  [request.session.application dismissKeyboard];
```

Comment:
I am resistant in landing stuff like this, because this will cause tons of issues complaining about it not working on iPhones. If you really need this. Please check for type of device and make the message super clear, that failure on iPhones is expected.